jans-cli-tui/cli_tui/jans_cli_tui.py

```
@@ -993,11 +992,23 @@ def get_plugin_by_id(self, pid: str) -> None:
             if plugin.pid == pid:
                 return plugin
 
+
+    def stay_in_config_menu(self, dialog=None):
```

Ruff (0.15.10)

[warning] Unused method argument: `dialog`

(ARG002)